Docker Build fails

**PassCore Server**
- OS: Linux
- Provider: LDAP
- Settings file (without sensitive information): 
- Log file (without sensitive information): 

**Describe the bug**
 I am not able to build the master branch

```
ERROR: unsatisfiable constraints:
  nodejs-10.19.0-r0:
    breaks: world[nodejs=10.14.2-r0]
    satisfies: npm-10.19.0-r0[nodejs]
The command '/bin/sh -c apk add --no-cache     icu-libs     nodejs=10.14.2-r0     nodejs-npm' returned a non-zero code: 2
```


**To Reproduce**
Steps to reproduce the behavior:
- git clone https://github.com/unosquare/passcore
- cd passcore
- docker build --rm -t passcore .

**Desktop (please complete the following information):**
 - OS: Linux

**Workaround**
Update nodejs to 10.19.0-r0

Now I have this error. Looks like nodejs is not installed.
```
/bin/sh: /tmp/tmpc842c5fccb6345598bd23ca158d54216.exec.cmd: line 2: node: not found
/build/src/Unosquare.PassCore.Web/Unosquare.PassCore.Web.csproj(54,5): warning MSB3073: The command "node --version" exited with code 127.
  /bin/sh: /tmp/tmpba28fb8e711446919275458169970417.exec.cmd: line 2: npm: not found
/build/src/Unosquare.PassCore.Web/Unosquare.PassCore.Web.csproj(57,5): error MSB3073: The command "npm install" exited with code 127.
The command '/bin/sh -c dotnet publish -c Release -o /publish /p:PASSCORE_PROVIDER=LDAP' returned a non-zero code: 1
```
It's in Step 7/12 : RUN dotnet publish -c Release -o /publish /p:PASSCORE_PROVIDER=LDAP

--- a/Dockerfile
+++ b/Dockerfile
@@ -4,7 +4,7 @@ FROM mcr.microsoft.com/dotnet/core/sdk:$DOTNET_VERSION-alpine3.9 AS build
 # Disable the invariant mode (set in base image)
 RUN apk add --no-cache \
     icu-libs \
-    nodejs=10.14.2-r0 \
+    nodejs=10.19.0-r0 \
     nodejs-npm
 
 WORKDIR /build

